cmd/lakectl/cmd/common_helpers.go

```diff
@@ -41,6 +43,8 @@ const (
 const internalPageSize = 1000          // when retreiving all records, use this page size under the hood
 const defaultAmountArgumentValue = 100 // when no amount is specified, use this value for the argument
 
+const defaultResponseOnSwaggerClient = http.StatusNoContent
```

Comment:
Prefer to use http.StatusNoContent when needed - the code will be readable without this indirection